lib/types/array.js

```diff
@@ -786,6 +786,28 @@ MongooseArray.mixin = {
       }
     }
     return -1;
+  },
+
+  /**
+   * Return whether or not the `obj` is included in the array.
+   *
+   * @param {Object} obj the item to check
+   * @return {Boolean}
+   * @api public
+   * @method includes
+   * @memberOf MongooseArray
+   */
+
+  includes: function indexOf(obj) {
```

Comment:
You forget to rename the function and I think it's better to directly return `this.indexOf(obj) !== -1` in this function for simplicity like `addToSet` and `pull` do.

Reply:
@Fonger updated, thanks!